internal/agent/device/hook/default_hooks.go

```diff
@@ -46,5 +56,15 @@ func defaultBeforeUpdateHooks() []v1alpha1.DeviceUpdateHookSpec {
 					"/var/run/flightctl/compose", "1m"),
 			},
 		},
+		{
+			Name:        util.StrToPtr("microshift manifest down"),
+			Path:        util.StrToPtr("/var/usr/klusterlet-manifests"),
+			Description: util.StrToPtr("Delete the provided microshift manifest"),
+			OnFile:      &[]v1alpha1.FileOperation{v1alpha1.FileOperationRemove},
+			Actions: []v1alpha1.HookAction{
+				marshalExecutable("kubectl delete -f {{ .FileName }}", &[]string{"KUBECONFIG=/var/lib/microshift/resources/kubeadmin/kubeconfig"},
```

Comment:
Are we 100% sure this is the desired behavior? 

Do we want to enable those hooks all by default? 

I think it's ok, although it'd be good to have a disable flag somehow for the defaults (just in case something goes wrong with our defaults)